app/main.py

```diff
@@ -1 +1,20 @@
-# write your code here
+import os
+import shutil
+
+
+def move_file(command: str) -> None:
+    parts = command.split()
+    if len(parts) != 3 or parts[0] != "mv":
+        raise ValueError("Invalid command format.")
```

Comment:
There is a syntax error here due to a missing closing parenthesis in the `ValueError` exception. Please add the missing parenthesis to fix this issue.